includes/view/User_view.php

```diff
@@ -677,11 +677,18 @@ function User_view(
             ]),
             div('row user-info', [
                 div('col-md-2', [
-                    heading(icon('phone')
-                        . '<a href="tel:' . $user_source->contact->dect . '">'
-                        . $user_source->contact->dect, 1)
-                        . '</a>'
-                ]),
+                    heading(
+                        icon('phone')
+                        . ' <a href="tel:' . $user_source->contact->dect . '">'
+                        . $user_source->contact->dect
+                        . '</a>', 1),
+                    heading(
```

Comment:
And a check for `user_messages` there would be great

Reply:
done